zeroex/orderwatch/order_watcher.go

```diff
@@ -392,7 +406,11 @@ func (w *Watcher) setupEventWatcher() {
 								w.handleDecodeErr(err, eventType)
 								continue
 							}
-							w.findOrderAndGenerateOrderEvents(exchangeFillEvent.OrderHash, log.TxHash)
+							orders = []*meshdb.Order{}
```

Comment:
`orders` has already been declared and initialized above, so we don't need to redeclare/re-initialize it.